src/test/java/org/opensrp/web/rest/PractitionerResourceTest.java

```diff
@@ -210,6 +210,29 @@ public void testUpdateWithJsonSyntaxException() throws Exception {
         verifyNoMoreInteractions(practitionerService);
     }
 
+	@Test
+	public void testGetPractitionersByPractitionerRoleIdentifierAndCode() throws Exception {
```

Comment:
Identation is different. This is a quick fix

Reply:
Updated